### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by marcioszka**
hey guys, checko out footer component. There are some exemplary references to pages with detailed information about our App, we may include modals (for cookies, terms&conditions and such).
I know our App is not going to be (any soon at least) big enough to have extended footer but it would be nice to add some more information than copyright note and about us 👯

### CODE REVIEW

File: src/components/footer/ReferenceLinks.ts
Comment (Zachiel) [inline src/components/footer/ReferenceLinks.ts:1]: Might consider extracting type definitions to a separate *.d.ts file, but it's not required with just 3 types as the file isn't that big.

File: src/components/footer/SplitLine.tsx
Comment (Zachiel) [inline src/components/footer/SplitLine.tsx:1]: You could make this a general utility component but it's not required anywhere yet. Just for future proofing reasons 😄

File: src/layouts/footer/FooterLayoutCss.scss [hunk 1]
```diff
@@ -0,0 +1,100 @@
+$primary-color: #fff;
+$link-color: white;
+$link-hover-color: pink;
+
+.footer {
+	background-color: #006400;
+	font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
```
Comment (Zachiel) [inline src/layouts/footer/FooterLayoutCss.scss]: Font faces are already declared in `index.css`:
```css
:root {
	font-family: 'Montserrat', 'Raleway', Helvetica, Arial, sans-serif;

	--console-font: 'Inconsolata', monospace;
	--handwriting-font: 'Caveat', cursive;
	--styled-font: 'Lobster', cursive;
	--fancy-3d-font: 'Nabla', cursive;
	text-align: center;
	font-size: 16px;
}
```
If any more are required add them there.

**Review by Zachiel — CHANGES_REQUESTED** [review]
Just change the font thingy in FooterLayoutCss.scss and it's good to go 😄

File: src/components/footer/SplitLine.tsx
Comment (marcioszka) [reply src/components/footer/SplitLine.tsx:1]: Yup, I use it in forms so it will be extracted :D

**Pull request merged**